pkg/mimirpb/compat.go

```diff
@@ -195,6 +238,106 @@ func fromSpansProtoToSpans(s []BucketSpan) []histogram.Span {
 	return spans
 }
 
+func FromHistogramToHistogramProto(timestamp int64, h *histogram.Histogram) Histogram {
```

Comment:
Shall we call this `FromIntegerHistogramToHistogramProto`?

Reply:
prometheus does not use that naming, let's leave it